engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -3869,4 +3869,22 @@ - (void)testInteractiveKeyboardShowKeyboardAndRemoveScreenshotAnimationIsReenabl
   [self waitForExpectations:@[ expectation ] timeout:10.0];
 }
 
+- (void)testCanPerformActionCaptureTextFromCamera {
```

Comment:
1. if you need to create a new test, please move it close to the context menu tests. 
2. Can you modify the existing test case related to context menu?